addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2110,6 +2110,16 @@ export class OdooEditor extends EventTarget {
     toolbarShow() {
         this._updateToolbar(true);
     }
+
+    closeDropDowns() {
+        this.toolbar.querySelectorAll("ul.dropdown-menu.show").forEach((e) => {
```

Comment:
For style coherence in the editor, we usually use `for...of` rather than `forEach` when it's gonna span several lines, and try to give meaningful names to variables:
```suggestion
        for (const dropdown of this.toolbar.querySelectorAll('ul.dropdown-menu.show')) {
```